docs/burial.rst

```diff
@@ -3,22 +3,42 @@ burial
 
 .. dfhack-tool::
     :summary: Allows burial in unowned coffins.
-    :tags: fort productivity buildings
+    :tags: fort | productivity | buildings
```

Comment:
```suggestion
    :tags: fort productivity buildings
```

Reply:
there's a required format for these tags so it can be read by the Sphinx documentation processing system